src/auth.ts

```diff
@@ -52,11 +53,30 @@ export const drive = google.drive({
  * Requests authorization to manage Apps Script projects.
  * @param {boolean} useLocalhost True if a local HTTP server should be run
  *     to handle the auth response. False if manual entry used.
+ * @param {any} creds either boolean or string. If true (boolean), then --creds
+ *     was used without any input file. Otherwise creds is a string of the file
+ *     they entered.
  */
-async function authorize(useLocalhost: boolean, writeToOwnKey: boolean) {
+async function authorize(useLocalhost: boolean, creds: any) {
+  let ownCreds = false;
+  creds = creds === true ? 'credentials.json' : creds;
```

Comment:
Never write `=== true`
Just `creds ? creds : '...'`;

Reply:
Strings evaluate to true so when they enter `--creds otherCredsFile.json` then we would rename it to `credentials.json` unfortunately. I see the other comments and there is probably a way to do this without using this or `any` so I'll look closer at the `commander` documentation.